### PR TITLE
MOSIP-34141

### PR DESCRIPTION
**Pull request by maheswaras**

### CODE REVIEW

File: .gitignore
Comment (mohanachandran-s) [inline .gitignore:1]: This file should not be deleted. undo it

**Review by mohanachandran-s — CHANGES_REQUESTED** [review]

File: inji-verify-test/pom.xml [hunk 1]
```diff
@@ -1,62 +1,127 @@
...
+		<dependency>
+			<groupId>org.apache.pdfbox</groupId>
+			<artifactId>pdfbox</artifactId>
+			<version>2.0.31</version>
+		</dependency>
+
+		<dependency>
+			<groupId>io.cucumber</groupId>
+			<artifactId>cucumber-java</artifactId>
+			<version>7.2.3</version>
```
[the remark this answers is not in the document]
Comment (maheswaras) [reply inji-verify-test/pom.xml]: Update

File: inji-verify-test/src/main/java/pages/BLE.java [hunk 2]
```diff
@@ -1,22 +1,21 @@
...
+
+
+public class BLE extends BasePage {
+	
+    private WebDriver driver;
+    public BLE(WebDriver driver) {
+        this.driver = driver;
+    }
+	public void ClickonBleTab() {
+		clickOnElement(driver, By.xpath("//button[@id='ble-tab']"));
```
[the remark this answers is not in the document]
Comment (maheswaras) [reply inji-verify-test/src/main/java/pages/BLE.java]: Updated

File: inji-verify-test/src/main/java/pages/ScanQRCodePage.java [hunk 3]
```diff
@@ -1,113 +1,115 @@
...
+
+public class ScanQRCodePage extends BasePage {
+	
+    private WebDriver driver;
+    public ScanQRCodePage(WebDriver driver) {
+        this.driver = driver;
+    }
+	
+	public void ClickonScanQRButtonTab() {
+		clickOnElement(driver, By.xpath("//*[@id='scan-qr-code-tab']"));
```
[the remark this answers is not in the document]
Comment (maheswaras) [reply inji-verify-test/src/main/java/pages/ScanQRCodePage.java]: Updated

File: .gitignore
Comment (maheswaras) [reply .gitignore:1]: Added file

**Review by mohanachandran-s — APPROVED** [review]

**Pull request closed without merging**